alacritty_terminal/src/config/window.rs

```diff
@@ -117,3 +141,45 @@ impl Dimensions {
         self.columns.0 as u32
     }
 }
+
+/// Window class
+#[derive(Clone, Default, Debug, Eq, PartialEq)]
+pub struct Class {
+    pub instance: Option<String>,
+    pub class: Option<String>
+}
+
+impl<'de> Deserialize<'de> for Class {
+    fn deserialize<D>(deserializer: D) -> Result<Class, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        let value = serde_yaml::Value::deserialize(deserializer)?;
+
+        match value {
+            Value::Null => Ok(Default::default()),
```

Comment:
I don't think there's a need to support an explicit `Null` here, is there? If this value is not present, it shouldn't be null but never call the deserializer, so this would probably just trigger for `value: Null`, right? So I think we can get rid of that.

Reply:
YAML has an explicit null (`null`).

Reply:
Yes, I was just trying to point out that we should not accept this explicit null. It should be treated like any other invalid value (like boolean, object, or whatever).